src/controllers/article.js

```diff
@@ -98,7 +98,7 @@ export const bookmarkArticle = async (req, res) => {
     }
     if (error.parent.file === 'uuid.c') {
       return res.status(409).json(errorResponseFormat({
```

Comment:
Nice work at making effort to return a friendly error message. Is there any reason why you are returning status 409 here?